CHANGELOG.md

```diff
@@ -3,11 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 5.0.3 - TBD
+## 6.0.0 - TBD
 ### Changed
 - `client.id` is now required to configure Kafka Receiver and Listener.
```

Comment:
`client.id` is no longer required right? For the receiver it looks like now we just treat it as another optional Kafka config property which will be handed over only if the user sets it?